src/Scaffolding/VS.Web.CG.Mvc/Templates/Blazor/Delete.tt

```diff
@@ -62,10 +60,10 @@
 </div>
 
 @code {
-    <#= modelName #>? <#= modelNameLowerInv #>;
+    private <#= modelName #>? <#= modelNameLowerInv #>;
 
     [SupplyParameterFromQuery]
-    public <#= primaryKeyShortTypeName #> <#= primaryKeyName #> { get; set; }
+    private <#= primaryKeyShortTypeName #> <#= primaryKeyName #> { get; set; }
```

Comment:
This raises a question for our general coverage ...

@MackinnonBuck ... When we placed our `[SupplyParameterFromQuery]` and `[SupplyParameterFromForm]` coverage, I placed examples with the `public` access modifier (and we don't directly comment on it ... it's just in our examples), and that seems like a mistake now. Should I have used `private` in keeping with our least accessible pattern? Is there ever a time that it makes sense to make these properties `public`?

Reply:
@guardrex, it used to be the case that `public` was required for `[SupplyParameterFromQuery]` and `[SupplyParameterFromForm]`. That's not the case anymore (as of .NET 8). There's not really a reason to make those `public`. Having a public `get` _might_ be useful in some cases if a parent component is interested in the property value, but there are better ways to flow information from a child component to a parent (e.g., event callbacks). A public `set` is probably not useful, as the provided value could just get overridden in the future by the form or query value (which could be `null` if the value doesn't exist).

I think it should be fine to change our examples to use `private` and not make a big deal about the reasoning. If anything, we could say something like "Unlike `[Parameter]` properties, properties annotated with `[SupplyParameterFromForm]` or `[SupplyParameterFromQuery]` aren't required to be public."

Reply:
Sounds good tyty